internal/activity/pusher.go

```diff
@@ -4,13 +4,14 @@ import (
 	"context"
 	"time"
 
-	"cdr.dev/coder-cli/coder-sdk"
 	"golang.org/x/time/rate"
 
+	"cdr.dev/coder-cli/coder-sdk"
+
 	"go.coder.com/flog"
 )
 
-const pushInterval = time.Minute
+const pushInterval = 1 * time.Minute
```

Comment:
I wouldn't do this.

Reply:
Can you extend on why? imho, it makes things easier to read and shows that it is a meant like the actual interval and not a unit

Reply:
I always assume that people typo'd `10*time.Minute` here, and think that time constants are clear enough.